src/commands/case_management.py

```diff
@@ -448,6 +448,41 @@ async def cmd_case_management_system(ctx: Context):
         await ctx.reply(f"{case.client}'s platform set to {case.platform.value}.")
 
 
+@command("quiet", require_channel=True, require_permission=RAT)
+async def cmd_case_management_quiet(ctx: Context):
+    # Check if there is an active rescue
+    """
+    if ctx.bot is not None:
+        if ctx.bot.board is not None:
+    """
+    for rescue in ctx.bot.board.values():  # type: Rescue
+        if rescue.active:
+            await ctx.reply("There is corrently an active rescue")
+            return
+
+    if ctx.bot.board._datetime_last_case == datetime.datetime.utcfromtimestamp(0):
+        await ctx.reply("Got no information yet")
+        return
+
+    timediff = divmod(
+        divmod(
+            int(
+                (
+                    datetime.datetime.now(tz=timezone.utc) - ctx.bot.board._datetime_last_case
```

Comment:
[error] access to protected data member of a different class

Reply:
Didn't consider it to be private, due to only having a single underscore. Default conventions ask for (at least) two underscores prefixing a private variable. Added an @Property to retrieve the value, as there are no external setters.